### PR TITLE
upgrade depedencies, add CI / tests, fixes #84

### PR DESCRIPTION
**Pull request by timkpaine**
Can see the action passing here: https://github.com/timkpaine/tree-finder/pull/1

### CODE REVIEW

File: packages/tree-finder/src/element/grid.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class TreeFinderGridElement<T extends IContentRow> extends RegularTableEl
       // data: object[][] -> array of arrays, each subarray containing all of the visible values for one column
       // .isBlank works around issue with formatting placeholder row that shows up when data is blank
       // TODO: refactor away .isBlank workaround
-      data: this.model.isBlank ? [Array(this.model.columns.length).fill("")] : this._getData(start_col, start_row, end_col, end_row),
+      data: this.model.isBlank ? Array(this.model.columns.length).fill([""]) : this._getData(start_col, start_row, end_col, end_row),
```
Comment (vidartf) [inline packages/tree-finder/src/element/grid.ts:91]: I see you found this as well! 👍

File: examples/simple/package.json [hunk 2]
```diff
@@ -21,13 +21,5 @@
   "dependencies": {
     "tree-finder": "^0.0.13",
     "tree-finder-mockcontents": "^0.0.13"
-  },
-  "devDependencies": {
```
Comment (vidartf) [inline examples/simple/package.json:25]: Just for my information: Why are we moving the dev dependencies to the central package.json?

**Review by vidartf — APPROVED** [review]
Looks good from a diff review, will test examples later.

File: examples/simple/package.json (hunk 2 above)
Comment (timkpaine) [reply examples/simple/package.json:25]: we'll probably use these ones across multiple subpackages

**Pull request closed without merging**